### DISCUSSION TITLE
[HELP] Installing Kura (ARM32 generic) in a Moxa (Debian9) device - Errors while setup with networking

### DISCUSSION THREAD

**Discussion by alexcorvis84** [eclipse-kura/kura#5327] [Q&A]
Hi @mattdibi @MMaiero ,

I have installed **kura_5.4.0_generic-arm32_installer.deb** in a Moxa UC-8112A-ME-T-LX device. It runs Debian GNU/Linux 9 (stretch), and network-manager is 1.6.2

I'm getting the following errors, which causes that **I can't access** to Network & Status options. All other settings and options work without problem.

![image](https://github.com/user-attachments/assets/981c69cb-0cdc-4ca3-b902-417d8524490b)
![image](https://github.com/user-attachments/assets/ffd77bfb-4bae-4cd0-9b7c-0a068cfc48d3)

If I try to access **Network or Status** tabs via web portal, a 500 Server Error is showed:

![image](https://github.com/user-attachments/assets/01e41e17-888d-4ba0-85de-52bc50f14344)

So, i guess these errors maybe are related to some incompatibility due network-manager version running on device?

I can´t upgrade it as it's on latest version available.

![image](https://github.com/user-attachments/assets/6e21d7c1-f62b-4398-9b67-5fa9da15d339)

How can I overcome these errors? I'd like to manage Networking from Kura portal

Any feedback is welcomed. Many thanks in advance!

**Comment by mattdibi** [comment]
Hello there!

From the log and the reported NetworkManager version I would say that this is a NetworkManager version problem. We developed our Kura-NetworkManager connector with much more recent NetworkManager's versions in mind (1.16+ IIRC). Kura is trying to call APIs that don't exits.

I would suggest you to try and update the NetworkManager's version. You can do so by adding the "bullseye" or "buster" repos to your machine and then proceed with the update of the package.

Alternatively you can update the whole OS which should ship with a more recent NetworkManager version
